.github/workflows/pull-request.yml

```
@@ -295,7 +295,33 @@ jobs:
         with:
           path: './coverage/lcov.info'
           min_coverage: 95.0
+  
+  Docker-Check:
+    needs: Test-Application
+    steps: 
+      - name: Check if Talawa API starts in Docker
+        run: |
+          docker-compose -f docker-compose.dev.yaml up -d --build
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)